### PR TITLE
ES2015ify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**
@sindresorhus @SamVerschueren @shinnn

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,121 +14,57 @@ $ npm install --save decompress
...
+#### options
 
-```js
-const Decompress = require('decompress');
+##### plugins
 
-new Decompress()
-	.use(Decompress.targz({strip: 1}));
-```
+Type: `array`<br>
```
Comment (sindresorhus) [inline readme.md]: array => Array

File: readme.md [hunk 2]
```diff
@@ -14,121 +14,57 @@ $ npm install --save decompress
...
+Default: `[decompressTar(), decompressTarGz(), decompressUnzip()]`
```
Comment (sindresorhus) [inline readme.md]: No gzip plugin defined as default: https://github.com/kevva/decompress/pull/38/files#diff-168726dbe96b3ce427e7fedce31bb0bcR68

**Comment by sindresorhus** [comment]
Nice work!

File: readme.md (hunk 2 above)
Comment (kevva) [inline readme.md]: I know, I haven't rewritten it yet :). I'm going to before merging this.

Comment (kevva) [inline readme.md]: Fixed in https://github.com/kevva/decompress/pull/38/commits/644d18f1690dd37649f6f92c440eb147741b3eb9.

**Pull request merged**